ward/testing.py

```diff
@@ -409,3 +454,16 @@ class TestResult:
     message: str = ""
     captured_stdout: str = ""
     captured_stderr: str = ""
+
+
+class Timer:
```

Comment:
The overall session run time is calculated using:

```python
from timeit import default_timer

start_run = default_timer()
# do stuff
time_taken = default_timer() - start_run
```

It'd be good if we kept the overall approach to timing consistent. Since there's now more than one thing being timed, it'd be nice if we could re-use the logic in both places.

Reply:
Totally agree that we should standardize, although I wonder if it should be on timeit.default_timer (or time.time) for that matter. The docs say that both measure wall-clock time instead of process time. I would think that we would want processs time, so we can be more independent the busyness of the server. But, I think pytest uses time.time(), so maybe it's not an issue.

I'll change it to timeit.default_timer, and maybe open another 'needs discussion' issue about
the time.